jekyll/_cci2/server/v4.3/air-gapped-installation/phase-1-prerequisites.adoc

```diff
@@ -86,19 +86,12 @@ cciserver.azurecr.io/nomad:1.1.2-74-7552671
 cciserver.azurecr.io/orb-service-analytics-migrator:0.1.64074-785435a1
 cciserver.azurecr.io/orb-service-migrator:0.1.64070-785435a1
 cciserver.azurecr.io/orb-service:0.1.64072-785435a1
-cciserver.azurecr.io/output-processing:0.1.66937-9609a831
 cciserver.azurecr.io/permissions-service-migrator:0.1.13413-18c6091
 cciserver.azurecr.io/permissions-service:0.1.13412-18c6091
-cciserver.azurecr.io/picard-dispatcher:0.1.65675-630fe5d4
 cciserver.azurecr.io/policy-service:0.1.5903-bf94009
-cciserver.azurecr.io/schedulerer:0.1.65674-630fe5d4
 cciserver.azurecr.io/server-license:0.1.113-be11908
 cciserver.azurecr.io/test-results:0.1.3402-e561bf7
 cciserver.azurecr.io/vault-cci:0.4.196-1af3417
-cciserver.azurecr.io/vm-gc:0.1.116274-09867360
```

Comment:
New ones need to be updated - but going to leave this for when list of other pieces is brought over

Reply:
We can use this to generate the list of images when we're ready https://github.com/circleci/server/tree/main/scripts/generate-images-script

Reply:
going to leave as other changes are going to happen with other teams and dont want to mess something up